akka-cluster-sharding/src/main/scala/akka/cluster/sharding/Shard.scala

```diff
@@ -265,6 +282,17 @@ private[akka] class Shard(
     }
   }
 
+  def passivateIdleEntities(): Unit = {
+    log.debug("Passivating idle entities")
```

Comment:
I don't think that log statement adds much value. More useful would be to collect the entityIds to be passivated and log something like 
```
log.debug("Passivating [{}] idle entities", idleEntities.size)
```

Reply:
I did it like this to not have to create an intermediate collection, and since each passivation also logs at debug, but maybe having to correlate this log entry with those is a bad idea. I'll change it.

Reply:
logging each passivation might be too much, that could be a storm of 1000s of log messages

Reply:
if someone need that they can add the logging in their actor

Reply:
hm, ok let's leave one of them "Entity stopped after passivation", but remove "Passivating started on entity"